python-requirements.txt

```
@@ -13,3 +13,6 @@ mako
 
 # Needed by dvsim.py (not actually used in Ibex)
 premailer
+
+# Needed by riscv-dv
+bitstring
```

Following @udinator's sugggestion, we can do that:

```suggestion
-r vendor/google_riscv-dv/requirements.txt
```

---

Oh, neat! I didn't know that pip did that. I'll push up an amended version.